src/data/gen2Banner-urls.ts

```diff
@@ -29,57 +57,196 @@ export const GEN2BANNER_URLS = [
+  '/nextjs/build-a-backend/graphqlapi/connect-to-api/',
   '/javascript/build-a-backend/graphqlapi/connect-existing-data/',
+  '/react/build-a-backend/graphqlapi/connect-existing-data/',
+  '/angular/build-a-backend/graphqlapi/connect-existing-data/',
+  '/vue/build-a-backend/graphqlapi/connect-existing-data/',
+  '/nextjs/build-a-backend/graphqlapi/connect-existing-data/',
   '/javascript/build-a-backend/graphqlapi/mutate-data/',
+  '/react/build-a-backend/graphqlapi/mutate-data/',
+  '/angular/build-a-backend/graphqlapi/mutate-data/',
+  '/vue/build-a-backend/graphqlapi/mutate-data/',
+  '/nextjs/build-a-backend/graphqlapi/mutate-data/',
   '/javascript/build-a-backend/graphqlapi/query-data/',
+  '/react/build-a-backend/graphqlapi/query-data/',
+  '/angular/build-a-backend/graphqlapi/query-data/',
+  '/vue/build-a-backend/graphqlapi/query-data/',
+  '/nextjs/build-a-backend/graphqlapi/query-data/',
   '/javascript/build-a-backend/graphqlapi/subscribe-data/',
+  '/react/build-a-backend/graphqlapi/subscribe-data/',
+  '/angular/build-a-backend/graphqlapi/subscribe-data/',
+  '/vue/build-a-backend/graphqlapi/subscribe-data/',
+  '/nextjs/build-a-backend/graphqlapi/subscribe-data/',
   '/javascript/tools/cli/graphqlapi/data-modeling/',
+  '/angular/tools/cli/graphqlapi/data-modeling/',
+  '/react/tools/cli/graphqlapi/data-modeling/',
+  '/nextjs/tools/cli/graphqlapi/data-modeling/',
+  '/vue/tools/cli/graphqlapi/data-modeling/',
   '/javascript/build-a-backend/graphqlapi/customize-authorization-rules/',
+  '/react/build-a-backend/graphqlapi/customize-authorization-rules/',
+  '/angular/build-a-backend/graphqlapi/customize-authorization-rules/',
+  '/vue/build-a-backend/graphqlapi/customize-authorization-rules/',
+  '/nextjs/build-a-backend/graphqlapi/customize-authorization-rules/',
   '/javascript/tools/cli/graphqlapi/custom-business-logic/',
+  '/angular/tools/cli/graphqlapi/custom-business-logic/',
+  '/react/tools/cli/graphqlapi/custom-business-logic/',
+  '/nextjs/tools/cli/graphqlapi/custom-business-logic/',
+  '/vue/tools/cli/graphqlapi/custom-business-logic/',
   '/javascript/tools/cli/graphqlapi/override-appsync/',
+  '/angular/tools/cli/graphqlapi/override-appsync/',
+  '/react/tools/cli/graphqlapi/override-appsync/',
+  '/nextjs/tools/cli/graphqlapi/override-appsync/',
+  '/vue/tools/cli/graphqlapi/override-appsync/',
   '/javascript/build-a-backend/graphqlapi/offline/',
+  '/react/build-a-backend/graphqlapi/offline/',
+  '/angular/build-a-backend/graphqlapi/offline/',
+  '/vue/build-a-backend/graphqlapi/offline/',
+  '/nextjs/build-a-backend/graphqlapi/offline/',
   '/javascript/build-a-backend/graphqlapi/best-practice/',
+  '/react/build-a-backend/graphqlapi/best-practice/',
+  '/angular/build-a-backend/graphqlapi/best-practice/',
+  '/vue/build-a-backend/graphqlapi/best-practice/',
+  '/nextjs/build-a-backend/graphqlapi/best-practice/',
   '/javascript/build-a-backend/graphqlapi/best-practice/deltasync/',
```

Comment:
All /graphqlapi/best-practice/deltasync/ references can be removed - including JS